tests/functional/simple_snapshot/test_hard_delete_snapshot.py

```diff
@@ -79,7 +94,7 @@ def test_snapshot_hard_delete(project):
     for result in snapshotted[10:]:
         # result is a tuple, the dbt_valid_to column is the latest
         assert isinstance(result[-1], datetime)
-        assert result[-1].replace(tzinfo=pytz.UTC) >= invalidated_snapshot_datetime
+        assert result[-1].replace(tzinfo=currenttz()) >= invalidated_snapshot_datetime
```

Comment:
this fixes an intermittent error that will show up in mine (and others, judging from slack threads) local test invocations. I ran into, and fixed, this as part of getting my local CI to precisely match our remote CI